kubernetes/cray-console-node/values.yaml

```diff
@@ -75,8 +76,8 @@ cray-service:
     log-forwarding:
       name: log-forwarding
       image:
-        repository: baseos/alpine
-        tag: 3.13.5
+        repository: alpine
+        tag: latest
```

Comment:
It is most convenient from an ease of upgrade standpoint to go with latest. However, I think it is ill-advised to do so from a stability standpoint. If the latest Alpine breaks something, then we have to figure out that the Alpine change is what broke us. We should pin to a specific trusted version.